internal/queryRunner.go

```diff
@@ -103,6 +104,7 @@ func (queryRunner *PgQueryRunner) getVersion() (err error) {
 
 // StartBackup informs the database that we are starting copy of cluster contents
 func (queryRunner *PgQueryRunner) StartBackup(backup string) (backupName string, lsnString string, inRecovery bool, dataDir string, err error) {
+	tracelog.InfoLogger.Println("Calling pg_start_backup()")
```

Comment:
InfoLogger -> DebugLogger

Reply:
Basically ok... just used INFO logger as @x4m mentioned that he had to support 20+ times users that got stuck here .. so sounds to me that it deserves INFO status ;-)

Reply:
Ok, so let's wait for him because I was not aware of this problem.

Reply:
That's true, the problem exists. Also, I think it will shed some light on what us happening.